package.json

```diff
@@ -33,7 +33,7 @@
     "lint-staged": "11.1.2",
     "prettier": "2.3.2",
     "rimraf": "3.0.2",
-    "ts-node": "10.1.0",
+    "ts-node": "10.2.0",
```

Comment:
I don't understand why we need ts-node here. It is only used by the `@coveo/search-token-server` package.